spring-integration-stomp/src/main/java/org/springframework/integration/stomp/AbstractStompSessionManager.java

```diff
@@ -111,14 +121,25 @@ public void setBeanName(String name) {
 		this.name = name;
 	}
 
+	public void setRecoveryInterval(int recoveryInterval) {
+		this.recoveryInterval = recoveryInterval;
+	}
+
 	@Override
 	public void afterPropertiesSet() throws Exception {
+		connect();
+	}
```

Comment:
Especially because we publish events and the event listener might not be started yet.